content/pt-br/docs/reference/node/_index.md

```diff
@@ -0,0 +1,16 @@
+---
+title: Informações de referência do Node
+weight: 80
+no_list: true
+---
+
+Esta seção contém os seguintes tópicos de referência sobre os nodes:
+
+* O [checkpoint API](/docs/reference/node/kubelet-checkpoint-api/) do kubelet
+
+* Uma lista de [Artigos sobre a remoção do dockershim e sobre o uso de tempos de execução compatíveis com o CRI](/docs/reference/node/topics-on-dockershim-and-cri-compatible-runtimes/)
```

Comment:
We have standardized the translation of "runtime" as "agente de execução" in Portuguese.